integration_tests/mockApis/findAndReferService.ts

```diff
@@ -6,7 +6,7 @@ export default {
     stubFor({
       request: {
         method: 'GET',
-        urlPattern: '/interventions\\?.*',
+        urlPattern: '/interventions.*',
```

Comment:
I think this will cause a problem when we implement the interventions details page, maybe something like this would work?
```suggestion
        urlPattern: '^/interventions/(community|custody|)$'
```